### ISSUE TITLE
PowerCLI module load fail to create PS drives

### ISSUE THREAD

**Issue by vGruntus** [PowerShell/PowerShellEditorServices#543]
When a script is loaded which will call functions from the VMware.PowerCLI module, the module fails to load correctly. This occurs only in the PowerShell Integrated Console which starts with the script, not the powershell.exe console. This does not occur if the PowerShell Integrated Console is started by opening a script, then closing the script and importing the module manually.
The error encountered is that the vis: and vmstores: PSDrives are not created. The following message is received:
`The specified mount name 'vis' is already in use.
The specified mount name 'vmstores' is already in use.`
Any script that references these drives will fail.

**Comment by tylerl0706** [comment]
@vGruntus Thanks for this! Sorry for the delay - It's been sometime since this issue was opened. Can you confirm it's still happening?

**Comment by dsolodow** [comment]
I can confirm that it is; I've encountered it as well.

**Comment by vGruntus** [comment]
@tylerl0706 Not a problem. Sorry I have not followed this up sooner, and that I've taken so long to respond. As mentioned by @dsolodow, this is still happening. If I load a PowerShell file into VSCode which contains PowerCLI components, the file loads successfully. When I hover the mouse over a PowerCLI cmdlet, VSCode will attempt to import the module and then produce the message above. This is occurring with the latest version of VSCode.

**Comment by tylerl0706** [comment]
So the workaround so far is 

import the module manually in the PowerShell Integrated Console before hovering over something in the script?

Or do you need to close the script first before the Integrated Console finishes starting up?

**Comment by rjmholt** [comment]
We had an issue for this in the vscode repo I think: https://github.com/PowerShell/vscode-powershell/issues/1383.

However, we decided that the bug lies in PowerCLI rather than in EditorServices -- there's nothing we can do to prevent non-idempotent drive creation on our side, since the module is farmed out to PSScriptAnalyzer.

Try this workspace configuration as a workaround:
```json
"powershell.scriptAnalysis.enable": false,
```

Closing as an external issue.
